src/engine/types/keys.rs

```diff
@@ -407,6 +409,13 @@ impl EncryptionInfo {
             .next()
             .cloned()
     }
+
+    /// Returns number of free token slots in the encryption information.
+    #[allow(clippy::cast_possible_truncation)]
```

Comment:
Can you move this allow so it is on top of the line that has the possible truncation, not the whole method?

Reply:
I tried doing that and when I did, Rust claimed that for some reason that invocation was unstable.